### DISCUSSION TITLE
Russian translation

### DISCUSSION THREAD

**Discussion by RACE4EVER** [mtkennerly/ludusavi#198] [Ideas]
I offer you my help in localising UI to Russian? if you need it)

**Comment by mtkennerly** [comment]
Hi! Thanks for offering to help :)

The easiest way to help with translations is via Crowdin:

* Ludusavi itself: https://crowdin.com/project/ludusavi (Russian is 10% translated)
* Ludusavi plugin for Playnite: https://crowdin.com/project/ludusavi-playnite (Russian is 0% translated)

It's broken up by message, so you can work on it piece by piece when you have time. Any help is very welcome!
